openupgrade_scripts/scripts/pos_hr/18.0.1.0/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,25 @@
+---Models in module 'pos_hr'---
+new model report.pos_hr.multi_employee_sales_report [abstract]
+new model report.pos_hr.single_employee_sales_report [abstract]
+# NOTHING TO DO: new feature added in https://github.com/odoo/odoo/commit/5193e458ac0dc165757fa0f3fc17796376ad1b3b
+
+---Fields in module 'pos_hr'---
+pos_hr       / account.bank.statement.line / employee_id (many2one)        : NEW relation: hr.employee
+# DONE: post-migration: Fill this field based on the employee related to the user who created the record.
```

Comment:
There can be multiple cashiers per user. Isn't there the possibility to obtain it from the `pos.order`?

Reply:
OK then.